Node.gitignore

```diff
@@ -100,10 +100,17 @@ dist
 # vuepress build output
 .vuepress/dist
 
+# vitepress build output
+**/.vitepress/dist
+
 # vuepress v2.x temp and cache directory
 .temp
 .cache
 
+# vitepress temp and cache directory
+**/.vitepress/temp
```

Comment:
I think this line is not needed.

Reply:
You're right.
In the *Tip* section of the docs [here](https://vitepress.dev/guide/getting-started#file-structure), it points to only `cache` and `dist` folders.